syntax/basic/function.vim

```diff
@@ -15,7 +15,7 @@ syntax match   typescriptAsyncFunc             contained /*/
   \ skipwhite skipempty
 
 syntax match   typescriptFuncName              contained /\K\k*/
-  \ nextgroup=@typescriptCallSignature
+  \ nextgroup=@typescriptCallSignature,typescriptTemplate
```

Comment:
I don't think FuncName is the correct choice here. But I need some more time to investigate how syntax highlight is done in your case. 

Would you like to revert this change?

Reply:
Hello, sorry yes I did add it to the wrong place.
This will not highlight the string when you have an object such as 
```
{
test: T`lkjsdf`,
}
```

I'll take another look and revert this one.